README.md

```diff
@@ -1,26 +1,44 @@
 # Coin CBC Rust bindings
 
-Rust bindings to the CoinOR CBC MILP Solveur using the C API.
+Rust bindings to the CoinOR CBC MILP Solver using the C API.
 
 Tested on Debian 10, AMD64, coinor-libcbc3 2.9.9+repack1-1.
+For more details on installing the `libCbc` dependencies, [see below](#prerequisites-installing-cbc-library-files).
 
 ## `coin_cbc_sys`
 
 This crate exposes raw bindings to the C functions.
 
 ## `coin_cbc`
 
-This crate expose safe rust bindings using
-`coin_cbc_sys`. `coin_cbc::raw::Model` exposes direct translation of
-the C function with assert to guaranty safe use. `coin_cbc::Model`
-exposes a more user friendly, rustic and efficient API: it was used
-successfully to solve MILP with 250,000 binary variables with
-unnoticeable overhead.
+This crate exposes safe rust bindings using `coin_cbc_sys`.
+`coin_cbc::raw::Model` exposes direct translation of the C function with assert to guaranty safe use.
+`coin_cbc::Model` exposes a more user friendly, rustic and efficient API: it was used successfully to solve MILP with 250,000 binary variables with unnoticeable overhead.
```

Comment:
Rewrapping paragraphes like that is quite anoying while reading at the diff. Better to keep as close as possible to the original line breaks for easy review.

Reply:
Should I redo this?

I try to stick to one sentence per row for better diffs during review. So this change is just out of that habit. But I guess you are right, it slightly pollutes the diff on this PR...

Reply:
As you wish. I can merge it like that or after you revert the line wrap.

Reply:
I also don't mind either way. So I guess the easiest is to just hit the merge button? ;)